sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -70,6 +79,7 @@ public sealed partial class ClientPipeline
     public partial class ClientPipelineOptions
     {
         public ClientPipelineOptions() { }
+        public System.ClientModel.Primitives.PipelinePolicy? LoggingPolicy { get { throw null; } set { } }
```

Comment:
Did we want to additionally have a `LoggingOptions` or `Sanitizer` property here?